standard/expressions.md

```diff
@@ -3214,95 +3214,95 @@ The predefined multiplication operators are listed below. The operators all comp
   
   <!-- Custom Word conversion: multiplication -->
   <table>
-  <!-- md equivalent:   ` `   | **`+y`**  | **`-y`**  | **`+0`**  | **`-0`**  | **`+∞`**  | **`-∞`**  | **`NaN`** -->
```

Comment:
This change must be reverted. See notes in the PR description for actions needed.

Reply:
I'll run two experiments:

1. Remove the MD equivalent comment. See if it round-trips.
2. Use a markdown table, and check the word and GH rendering.

Reply:
OK. I ran the markdown table experiment. That works, and the output is better.  

I think this is the solution we should use. I'm wondering if we should remove the code fences around these tables. It looks kind of ugly.

Reply:
Speculation from looking at the table source: It was not correctly formed markdown and that caused issues with our other tools.

The badly formed parts were that the table was missing the opening `|` on the first column, and the closing `|` on the last column. The first row wasn't formatted as a header row. (Meaning the 2nd row wasn't specifying the justification).  

In time, it may be worth looking at the other tables where we have HTML and see if we can use properly formatted markdown tables for those as well.

Reply:
@BillWagner These tables are getting flagged in the build report for dotnet/docs because there's a column without a heading. You can resolve it by bolding everything in the first column to show that it's a data matrix.

https://buildapi.docs.microsoft.com/Output/PullRequest/a3bda507-3390-de91-8f7f-26f90f4e5fc8/202206090300048116-29770/BuildReport?accessString=7e8b720a7ea2f64577ecd0387de1f9e8f766ed8d1f35267ed986663930190107

Reply:
Thanks. I'll handle this as part of #585, which I'm planning on this week.